File: birdmeal/src/main/java/com/backend/birdmeal/dto/SellerDto.java
package com.backend.birdmeal.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SellerDto {

    private long sellerSeq;
    private String sellerEmail;
    private String sellerNickname;
    private String sellerEoa;
    private String sellerTel;
    private String sellerAddress;
    private String sellerInfo;
    private String sellerCreateDate;
    private String sellerUpdateDate;

}

File: AOS/BirdMeal/app/src/main/java/com/ssafy/birdmeal/model/dto/SellerDto.kt
package com.ssafy.birdmeal.model.dto

import com.google.gson.annotations.SerializedName

data class SellerDto(
    @SerializedName("sellerSeq") val sellerSeq: Int,
    @SerializedName("sellerImg") val sellerImg: String,
    @SerializedName("sellerWallet") val sellerWallet: String,
    @SerializedName("sellerAddress") val sellerAddress: String,
    @SerializedName("sellerEmail") val sellerEmail: String,
    @SerializedName("sellerEoa") val sellerEoa: String,
    @SerializedName("sellerInfo") val sellerInfo: String? = "",
    @SerializedName("sellerNickname") val sellerNickname: String,
    @SerializedName("sellerTel") val sellerTel: String,
    @SerializedName("sellerCreateDate") val sellerCreateDate: String,
    @SerializedName("sellerUpdateDate") val sellerUpdateDate: String
)